src/services/addressService/index.js

```diff
@@ -3,5 +3,6 @@ import CreateByZipCodeWorkflow from './CreateByZipCodeWorkflow';
 export default {
   workflows: {
     createByZipCode: new CreateByZipCodeWorkflow().handler,
+    createByAddress: new CreateByZipCodeWorkflow().handler,
```

Comment:
```suggestion
    create: new CreateWorkflow().handler,
```
A gente precisou especificar o 'ByZipCode' pra diferenciar da criação manual. Como esse endpoint é uma criação normal a gente pode deixar somente `create` e o nome da classe `CreateWorkflow`